modules/javafx.graphics/src/main/java/com/sun/glass/utils/NativeLibLoader.java

```diff
@@ -29,14 +29,21 @@
 import java.security.AccessController;
 import java.security.PrivilegedAction;
 import java.util.HashSet;
+import java.io.*;
+import java.nio.*;
+import java.nio.file.*;
```

Comment:
Can you expand these (we generally avoid wildcard imports)

Reply:
done